### ISSUE TITLE
Harvard: Unknown files in list of active processing jobs

### ISSUE THREAD

**Issue by cbaksik** [pod4lib/aggregator#610]
There are some files listed in the Resource column of our Active processing jobs, stuck in the MARC profiling job, which are not files we submitted. I'm not sure what these are. They are all 5-digit files names with no extensions. 

![image](https://user-images.githubusercontent.com/788203/163590213-fbc8ed2c-e1a7-4430-85c3-ba743aa2d207.png)

**Comment by cbaksik** [comment]
@Hurlyburly2 @ktamaral

**Comment by corylown** [comment]
@cbaksik I think these particular jobs are leftover from before we fixed a bug in the file identification code. After we fixed that bug I re-analyzed all Harvard's files, which may have broken the association between these job trackers and the files they were working on (that's my guess anyway). The numbers are the database identifiers for the files. I've deleted these job trackers so they should no longer appear in the list of jobs.
